torch/csrc/Module.cpp

```diff
@@ -683,6 +687,15 @@ void initModule(PyObject *module);
 }} // namespace torch::cuda
 #endif
 
+#ifdef USE_MLCOMPUTE
+PyMethodDef* THCPModuleMLC_methods();
+namespace torch { namespace mlc {
+
+void initBindings(PyObject *module);
+
+}} // namespace torch::mlc
```

Comment:
Why those needs to be defined here, rather than in `MLCInit.h`?

Reply:
We were trying to follow the same pattern as CUDA. However, we can restructure it so some of these can be moved directly onto MLCInit.h.